src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write code here
+function sortList() {
+  const list = [...document.querySelectorAll('ul li')];
+
+  list.sort((a, b) => {
+    const item1 = parseInt(a.dataset.salary.replace(/[\D]/g, ''));
+    const item2 = parseInt(b.dataset.salary.replace(/[\D]/g, ''));
+
+    return item2 - item1;
+  });
+
+  list.forEach((item) => {
+    document.querySelector('ul').appendChild(item);
+  });
+}
+
+function getEmployees() {
+  const list = [...document.querySelectorAll('ul')];
+
+  return list.map((item) => ({
+    name: item.textContent,
```

Comment:
This line is using 'textContent' to get the name, which assumes the 'li' element's text is the employee's name. Ensure this matches the HTML structure you are working with.